Doc/faq/general.rst

```diff
@@ -312,13 +312,13 @@ guaranteed that interfaces will remain the same throughout a series of bugfix
 releases.
 
 The latest stable releases can always be found on the `Python download page
-<https://www.python.org/downloads/>`_.  There are two production-ready version
-of Python: 2.x and 3.x, but the recommended one at this times is Python 3.x.
-Although Python 2.x is still widely used, `it will not be
-maintained after January 1, 2020 <https://www.python.org/dev/peps/pep-0373/>`_.
-Python 2.x was known for having more third-party libraries available, however,
-by the time of this writing, most of the widely used libraries support Python 3.x,
-and some are even dropping the Python 2.x support.
+<https://www.python.org/downloads/>`_.  There are two production-ready versions
+of Python: 2.x and 3.x, with the recommended one at this time being 3.x.
+Although 2.x is still widely used, `it will not be maintained after
+January 1, 2020 <https://www.python.org/dev/peps/pep-0373/>`_.  2.x was known
```

Comment:
I think I'd drop the historical "2.x used to be better" wording, and just make this "Python 3.x is supported by most widely used libraries".